### PR TITLE
Update README on python version and dabase setup

### PR DESCRIPTION
**Pull request by Nuwini**
I added the python version to use under the installation as well as database setup steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,30 @@ Automated Download Queue for Enterprise to take the best use of Internet bandwid
...
+
+1. Type below in a MySql terminal.
+
+```
+create database Bassa
+```
+
+2. Then type below in the root of project.
+
+````
```
Comment (saisankargochhayat) [inline README.md]: use -'''

Comment (saisankargochhayat) [inline README.md]: Remove the extra single quote.

Comment (Nuwini) [inline README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -9,11 +9,30 @@ Automated Download Queue for Enterprise to take the best use of Internet bandwid
...
 ```
+## Database Setup
+
+1. Type below in a MySql terminal.
+
+```
+create database Bassa
+```
+
+2. Then type below in the root of project.
```
Comment (rajikaimal) [inline README.md]: Maybe you can get rid of `then` word in 2 & 3.

Comment (Nuwini) [inline README.md]: Updated.

**Pull request merged**